docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+services:
+  bot-detector-web:
+    build: .
+    container_name: bot-detector-web
+    volumes:
+      - /config:/usr/share/nginx/html/config
+    ports:
+      - 8080:8080
+    restart: unless-stopped
```

Comment:
missing other services & network
note fix indentation of below & add target if we do the dockerfile
```
version: '3.4'
services:
  mysql:
    build:
      context: ../bot-detector-mysql
      dockerfile: Dockerfile
    image: bot-detector/bd-mysql:latest
    environment:
      - MYSQL_ROOT_PASSWORD=root_bot_buster
      - MYSQL_USER=botssuck
      - MYSQL_PASSWORD=botdetector
    volumes:
      - '../bot-detector-mysql/mount:/var/lib/mysql'
      - '../bot-detector-mysql/docker-entrypoint-initdb.d/:/docker-entrypoint-initdb.d/'
    ports:
      - "3306:3306"
    networks:
      - botdetector-network

  api:
    build:
      context: ../Bot-Detector-Core-Files
      dockerfile: Dockerfile
      args:
        root_path: ''
        api_port: 5000
    image: bot-detector/bd-api:latest
    environment:
      - sql_uri=mysql+asyncmy://root:root_bot_buster@mysql:3306/playerdata
      - discord_sql_uri=mysql+asyncmy://root:root_bot_buster@mysql:3306/discord
      - token=verify_ban
    volumes:
      - '../Bot-Detector-Core-Files/api:/code/api'
    ports:
      - "5000:5000"
    networks:
      - botdetector-network
    depends_on:
      - mysql
bot-detector-web:
    build: .
    container_name: bot-detector-web
    volumes:
      - /config:/usr/share/nginx/html/config
    ports:
      - 8080:8080
    restart: unless-stopped
networks:
  botdetector-network:
```